packages/e2e-test-utils-playwright/src/posts.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Delete all posts using REST API.
+ *
+ * @this {import('.').TestUtils}
+ */
+export async function deleteAllPosts() {
```

Comment:
This util is originally `trashAllPosts()`, altered to use `rest` to speed it up.